pkg/watch/registrar.go

```diff
@@ -93,7 +94,7 @@ func (r *recordKeeper) RemoveRegistrar(parentName string) error {
 	return nil
 }
 
-func (r *recordKeeper) Update(parentName string, m vitalsByGVK) {
+func (r *recordKeeper) Update(ctx context.Context, parentName string, m vitalsByGVK) {
```

Comment:
It looks like this context is no longer used?

Reply:
Done.